invenio_deposit/views/ui.py

```diff
@@ -68,12 +76,37 @@ def index():
         return render_template(current_app.config['DEPOSIT_UI_INDEX_TEMPLATE'])
 
     @blueprint.route('/deposit/new')
+    @blueprint.route('/deposit/new/<string:deposit_type>')
```

Comment:
@pamfilos @egabancho Maybe we can register it **only if** there are more than one deposit types.